src/town_cmd.cpp

```diff
@@ -454,13 +454,22 @@ uint32 GetWorldPopulation()
 
 /**
  * Remove stations from nearby station list if a town is no longer in the catchment area of each.
- * @param t Town to work on
+ * To improve performance only checks stations that cover the provided house area (doesn't need to contain an actual house).
+ * @param t The town to work on
+ * @param tile The location of house area (north part)
+ * @param size The size of house area
  */
-static void RemoveNearbyStations(Town *t)
+static void RemoveNearbyStations(Town *t, TileIndex tile, BuildingFlags size)
```

Comment:
perhaps `sizeflags` ? `size` indicates a measurement to me

Reply:
or `flags`, but yes `size` feels wrong